packaging/amazon-linux-ami/ecs.service

```diff
@@ -0,0 +1,31 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the
+# "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and
+# limitations under the License.
+
+[Unit]
+Description=Amazon Elastic Container Service - container agent
+Documentation=https://github.com/aws/amazon-ecs-agent
+Requires=docker.service
+After=docker.service
+After=cloud-final.service
+
+[Service]
+Type=simple
+Restart=on-failure
+RestartPreventExitStatus=5
+RestartSec=10
+EnvironmentFile=-/etc/ecs/ecs.config
```

Comment:
is the hyphen `-` needed before the config file path?

Reply:
The hyphen allows for the service to continue in case there is no environment file. The config file is not set up by default so I figured this would be useful in the case that a user did not create a `ecs.config` file

Reply:
fwiw, this is what we are currently doing: https://github.com/aws/amazon-ecs-init/blob/master/packaging/amazon-linux-ami/ecs.service#L27